yang-semver/ietf-yang-semver.yang

```diff
@@ -16,16 +16,16 @@ module ietf-yang-semver {
 
      Author:   Joe Clarke
                <mailto:jclarke@cisco.com>
-     Author:   Benoit Claise
-               <mailto:benoit.claise@huawei.com>
-     Author:   Reshad Rahman
-               <mailto:reshad@yahoo.com>
      Author:   Robert Wilton
                <mailto:rwilton@cisco.com>
+     Author:   Reshad Rahman
+               <mailto:reshad@yahoo.com>
      Author:   Balazs Lengyel
                <mailto:balazs.lengyel@ericsson.com>
      Author:   Jason Sterne
-               <mailto:jason.sterne@nokia.com>";
+               <mailto:jason.sterne@nokia.com>
```

Comment:
Another nit: not sure if we're supposed to change the revision date for such a change

Reply:
I think you're right. I just updated the PR for this.

Reply:
If the YANG module is not touched, we should not be changing revision info.  But in this case, we are adjusting content, so I think this change is correct.